### PR TITLE
[EchoCallset] Retry with more memory VCF extract [VS-1351]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

**Review by rsasch — COMMENTED** [review]
link to test run?

**Review by gbggrant — APPROVED** [review]

File: scripts/variantstore/wdl/GvsExtractCallset.wdl [hunk 1]
```diff
@@ -371,7 +371,18 @@ task ExtractTask {
...
+    # https://support.terra.bio/hc/en-us/articles/4403215299355-Out-of-Memory-Retry
+    if [[ ${MEM_UNIT} == "GB" ]]
+    then
+        memory_mb=$(python3 -c "from math import floor; print(floor((${MEM_SIZE} - 3) * 1000))")
+    else
+        echo "Unexpected memory unit: ${MEM_UNIT}" 1>&2
+        exit 1
+    fi
+
+    gatk --java-options "-Xmx${memory_mb}m" \
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:385]: oooooh i didn't know we could do this!
(can we do this all over the VAT pipeline too!??!?!)

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/wdl/GvsExtractCallset.wdl (hunk 1 above)
Comment (mcovarr) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:385]: Sure why not. The hard part is remembering to tick the 'Retry with more memory' option when launching the workflow lol

Comment (rsasch) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:385]: And we already need to remember to click on the reference disk checkbox for that...

**Comment by mcovarr** [comment]
[Intentionally failing test run here](https://job-manager.dsde-prod.broadinstitute.org/jobs/5d936172-f29c-4a59-ada5-e55883beaaf9) where I sabotaged the code to simulate an out of memory condition. You can see on the first attempt memory usage looks like

```
echo 'MEM_SIZE: 40.0, MEM_UNIT: GB, memory_mb: 37000, memory_gib: 40'
```

after three retry-with-more-memory retries, things look like:

```
echo 'MEM_SIZE: 69.12, MEM_UNIT: GB, memory_mb: 66120, memory_gib: 40'
```

**Pull request merged**